coordinator/core/marbleapi.go

```diff
@@ -88,6 +90,24 @@ func (c *Core) Activate(ctx context.Context, req *rpc.ActivationReq) (*rpc.Activ
 	}
 
 	marble := c.manifest.Marbles[req.GetMarbleType()] // existence has been checked in verifyManifestRequirement
+	// add TTLS config to Env
+	ttlsConf := make(map[string]map[string]string)
+	ttlsConf["tls"] = make(map[string]string)
+	for _, value := range marble.TLS {
+		for _, value2 := range c.manifest.TLS[value].Outgoing {
+			pemCert := pem.Block{Type: "CERTIFICATE", Bytes: c.rootCert.Raw}
```

Comment:
```suggestion
			pemCert := pem.Block{Type: "CERTIFICATE", Bytes: c.intermediateCert.Raw}
```
Marbles use an intermediate cert as root. This is regenerated on a manifest update. Connections between marbles then intentionally fail until the marble is restarted and attested based on the new manifest.